backend/src/controllers/MeetingController.ts

```diff
@@ -97,12 +97,89 @@ const updateMeeting = asyncHandler(
  * @desc 	Delete a specific meeting
  * @route 	DELETE /api/meetings/:meetingId
  */
-const deleteMeeting = asyncHandler(async (req: TypedRequestParams<MeetingIdParam>, res: Response) => {
+ const deleteMeeting = asyncHandler(async (req: TypedRequestParams<MeetingIdParam>, res: Response) => {
     const meeting = await Meeting.findByIdAndDelete(req.params.meetingId);
     if (!meeting) {
         return res.notFound(`There is no meeting with the id ${req.params.meetingId}`);
     }
     res.sendStatus(204);
 });
+/**
+ * @desc 	Update the absence of a user for a meeting
+ * @route 	POST /api/meetings/:meetingId/attendance/going
+ */
+ const updateAbsence = asyncHandler(async (req: TypedRequest<UpdateAbsenceRequest, MeetingIdParam>, res) => {
+    const meeting = await Meeting.findById(req.params.meetingId);
+    if (!meeting) {
+        return res.notFound(`There is no meeting with id ${req.params.meetingId}`);
+    }
+    const currentAttendance = meeting.attendance.get(req.body.requester._id);
+
```

Comment:
Try to use `req.body.requester.id` where possible. It's the same as `_id` for users, but it's nice to be consistent.